packages/core/src/odata/v4/selectable/collection-field.ts

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+import { Entity } from '../entity';
+import {
+  ComplexTypeField,
+  Field,
+  SelectableEdmTypeField,
+  SimpleTypeFields
+} from '../../common/selectable';
+import { Constructable } from '../../common';
+
+export class CollectionField<EntityT extends Entity> extends Field<EntityT>
```

Comment:
Maybe we could also implement the [`in`](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_In) function (and add a test for it). This would help understand whether we have thought collections through for filters as well. This can also be done in a follow up, but as far as I can see this should be very simple and if it is not it would be good to understand this early on.

Reply:
- `in` function should be called from simple field e.g., `City` as string field -> `Address/City in ('Redmond', 'London')`
- `eq`/`ne` don't support collection field, since the reference service says:`The operand for a binary operator 'Equal' is not a single value. Binary operators require both operands to be single values.`

Reply:
As discussed ;)